src/av2/geometry/utm.py

```diff
@@ -0,0 +1,105 @@
+# <Copyright 2022, Argo AI, LLC. Released under the MIT license.>
+
+"""Utilities for converting AV2 city coordinates to UTM or WGS84 coordinate systems."""
+
+from enum import unique, Enum
+from typing import Dict, Final, Tuple, Union
+
+import numpy as np
+from pyproj import Proj
+
+from av2.utils.typing import NDArrayFloat, NDArrayInt
+
+
+@unique
+class CityName(str, Enum):
+    """Abbreviations of names of cities featured in Argoverse 2."""
+
+    ATX = "ATX"  # Austin, Texas
+    DTW = "DTW"  # Detroit, Michigan
+    MIA = "MIA"  # Miami, Florida
+    PAO = "PAO"  # Palo Alto, California
+    PIT = "PIT"  # Pittsburgh, PA
+    WDC = "WDC"  # Washington, DC
+
+
+# All are North UTM zones (Northern hemisphere)
+UTM_ZONE_MAP: Final[Dict[CityName, int]] = {
+    CityName.ATX: 14,
+    CityName.DTW: 17,
+    CityName.MIA: 17,
+    CityName.PAO: 10,
+    CityName.PIT: 17,
+    CityName.WDC: 18,
+}
+
+
+# as (lat, long) tuples
+CITY_ORIGIN_LATLONG_DICT: Final[Dict[CityName, Tuple[float, float]]] = {
+    CityName.ATX: (30.27464237939507, -97.7404457407424),
+    CityName.DTW: (42.29993066912924, -83.17555750783717),
+    CityName.MIA: (25.77452579915163, -80.19656914449405),
+    CityName.PAO: (37.416065, -122.13571963362166),
+    CityName.PIT: (40.44177902989321, -80.01294377242584),
+    CityName.WDC: (38.889377, -77.0355047439081),
+}
+
+
+def convert_gps_to_utm(lat: float, long: float, city_name: CityName) -> Tuple[float, float]:
```

Comment:
I understand that `long` is common in the GIS community, but I think this is a bit confusing with the numerical type, `long`. Perhaps we could use `lon`?

Reply:
Good call. I switched to `longitude`, since I think it's not much longer, and may be clearer.